modules/site-health/audit-autoloaded-options/load.php

```diff
@@ -117,3 +117,44 @@ function perflab_aao_autoloaded_options_size() {
 	global $wpdb;
 	return (int) $wpdb->get_var( 'SELECT SUM(LENGTH(option_value)) FROM ' . $wpdb->prefix . 'options WHERE autoload = \'yes\'' );
 }
+
+/**
+ * Fetch autoload top list.
+ *
+ * @since n.e.x.t
+ *
+ * @return array autoloaded data - autoload size, entries, option and its names.
+ */
+function perflab_aao_autoloaded_options() {
+	global $wpdb;
+
+	/**
+	 * Filters max bytes threshold to get options from Database.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param int $option_threshold Autoloaded options threshold size. Default 100.
+	 */
+	$option_threshold = apply_filters( 'perflab_aao_autoloaded_option_table_threshold', 100 );
+
+	return $wpdb->get_results( $wpdb->prepare( "SELECT option_name, LENGTH(option_value) AS option_value_length FROM `wp_options` WHERE autoload='yes' AND LENGTH(option_value) > %d ORDER BY option_value_length DESC LIMIT 20", $option_threshold ) );
+}
+
+/**
+ * Get formatted autoload options table.
+ *
+ * @since n.e.x.t
+ *
+ * @return string HTML formatted table.
+ */
+function perflab_aao_autoloaded_options_table() {
```

Comment:
See above, I think `perflab_aao_get_autoloaded_options_table()` would be a more descriptive name. Otherwise, you don't know whether the function returns the table or renders it directly.